### PR TITLE
Replace MongoDBLibrary with util/mongoapi

### PR DESCRIPTION
**Pull request by gatici**
This PR replaces MongoDBLibrary with util/mongoapi.

### CODE REVIEW

File: backend/webui_service/webui_init.go [hunk 1]
```diff
@@ -161,14 +163,15 @@ func (webui *WEBUI) Start() {
...
+		// Create the MongoDB Client
+		var c, _ = mongoapi.NewMongoClient(mongodb.Url, mongodb.Name)
 		for {
-			if MongoDBLibrary.Client != nil {
+			if c.Client != nil {
+				commonDBClient.Client = c.Client
 				break
 			} else {
-				MongoDBLibrary.SetMongoDB(mongodb.Name, mongodb.Url)
+				c, _ = mongoapi.NewMongoClient(mongodb.Url, mongodb.Name)
```
Comment (thakurajayL) [inline backend/webui_service/webui_init.go:173]: Should have some timer. Else this will be in tight loop.

Comment (gatici) [reply backend/webui_service/webui_init.go]: We add a timeout here.

File: backend/webui_service/webui_init.go [hunk 2]
```diff
@@ -161,15 +163,25 @@ func (webui *WEBUI) Start() {
 	if factory.WebUIConfig.Configuration.Mode5G {
 		// get config file info from WebUIConfig
 		mongodb := factory.WebUIConfig.Configuration.Mongodb
-
 		// Connect to MongoDB
-		MongoDBLibrary.SetMongoDB(mongodb.Name, mongodb.Url)
+		ticker := time.NewTicker(1 * time.Second)
+		defer func() { ticker.Stop() }()
+		timer := time.After(10 * time.Second)
+	ConnectMongo:
```
Comment (thakurajayL) [inline backend/webui_service/webui_init.go:170]: I think we can write it efficient way?

File: backend/webui_service/webui_init.go [hunk 3]
```diff
@@ -161,15 +163,25 @@ func (webui *WEBUI) Start() {
...
 		for {
-			if MongoDBLibrary.Client != nil {
-				break
-			} else {
-				MongoDBLibrary.SetMongoDB(mongodb.Name, mongodb.Url)
-				continue
+			select {
+			case <-ticker.C:
```
[the remark this answers is not in the document]
Comment (gatici) [reply backend/webui_service/webui_init.go:184]: This code is modified to start immediately.

**Comment by gatici** [comment]
https://github.com/omec-project/webconsole/pull/127 is addressing the change.

**Pull request closed without merging**